source/includes/administration/_service_connections.md

```diff
@@ -59,3 +59,92 @@ Attributes | &nbsp;
 `name`<br/>*string* | The name of the service connection
 `type`<br/>*string* | The type of the service connection.
 `status`<br/>*Object* | Status of the service connection. Tells you if the service is up.<br/>*includes*: `lastUpdated`, `reachable`
+
+<!-------------------- GET APIINFO -------------------->
+
+### Retrieve API credentials
+
+`GET /services/connections/:id/apiInfo`
+
+```shell
+# Retrieve your API credentials
+curl -X GET \
+  "https://system.cloudmc-dev.cloudops-devteam.com/rest/services/connections/[id]/apiInfo?environmentId=[id]" \
+  -H "MC-Api-Key: your_api_key" \
+# Response body example
+```
+```json
+{
+    "data": {
+        "canRegenerateCredentials": true,
+        "fields": [{
+            "key": "endpoint",
+            "value": "https://your_endpoint.com/auth",
+            "sensitive": false
+        }, {
+            "key": "api_key",
+            "value": "my_api_key",
+            "sensitive": true
+        }, {
+            "key": "secret_key",
+            "value": "my_secret_key",
+            "sensitive": true
+        }, {
+            "key": "project_id",
+            "value": "079bdead-61b5-4b38-86ed-dbbf963808ec",
+            "sensitive": false
+        }]
+    }
+}
+```
+
```

Comment:
You are missing something like this here:

`<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/apiInfo</code>`

Reply:
Spoke with Simon, we can leave the request path as `GET /services/connections/:id/apiInfo` (see line 67)